lib/mobius/cogs/help.ex

```diff
@@ -0,0 +1,123 @@
+defmodule Mobius.Cogs.Help do
+  @moduledoc "The help command's cog"
+
+  use Mobius.Cog
+
+  alias Mobius.CogUtils
+  # Unsafe to use for 3rd party cogs
+  alias Mobius.Core.Cog
+  alias Mobius.Core.Command
+  alias Mobius.Services.CogLoader
+
+  @help_footer """
+  Type `[p]help Cog` for help about a specific cog
+  Type `[p]help command` for help about a specific command
+  """
+
+  @cog_footer """
+  Type `[p]help command` for help about a specific command
+  """
+
+  @not_found """
+  Cog or command not found. Use `[p]help` for a list of cogs and commands.
+  """
+
+  @doc "This command"
+  command "help" do
+    CogLoader.list_cogs()
+    |> format_cogs()
+    |> reply()
+  end
+
+  @doc """
+  Shows help for a cog or a command
+
+  The name is case sensitive to distinguish between cog names and command names.
+  For example `[p]help Help` shows the cog, but `[p]help help` shows the command.
+  """
+  command "help", cog_or_command_name: :string do
+    cogs = CogLoader.list_cogs()
+
+    cog_or_command_name
+    |> try_cog(cogs)
+    |> try_command(cog_or_command_name, cogs)
+    |> reply()
+  end
+
+  defp try_cog(part, cogs) do
+    case find_cog(part, cogs) do
+      %Cog{} = cog -> format_specific_cog(cog)
+      _ -> nil
+    end
+  end
+
+  defp try_command(content, _part, _cogs) when is_binary(content), do: content
+
+  defp try_command(nil, part, cogs) do
+    case find_command(part, cogs) do
+      arities when is_map(arities) -> format_specific_command(arities)
+      _ -> nil
+    end
+  end
+
+  defp format_specific_cog(%Cog{} = cog) do
+    commands = list_cog_commands(cog)
+    command_list = CogUtils.format_categories_list([{"Commands", commands}])
+
+    "#{cog.description}\n```#{command_list}```#{@cog_footer}"
```

Comment:
```suggestion
    formatted_commands = CogUtils.format_categories_list([{"Commands", commands}])

    "#{cog.description}\n```#{formatted_commands}```#{@cog_footer}"
```
I feel like "list" could be misleading here